app/workers/teacher_training_public_api/sync_all_providers_and_courses_worker.rb

```diff
@@ -4,11 +4,27 @@ class SyncAllProvidersAndCoursesWorker
 
     sidekiq_options retry: 3, queue: :low_priority
 
-    def perform(incremental = true, year = ::RecruitmentCycle.current_year, suppress_sync_update_errors = false)
+    def perform(incremental = true, year = nil, suppress_sync_update_errors = false)
       return if HostingEnvironment.review?
 
+      year ||= year_to_sync
+
       SyncSubjects.new.perform
       SyncAllProvidersAndCourses.call(recruitment_cycle_year: year, incremental_sync: incremental, suppress_sync_update_errors: suppress_sync_update_errors)
+
+      if FeatureFlag.active?(:sync_next_cycle) && !CycleTimetable.find_down?
```

Comment:
This might be better as:
```suggestion
      if FeatureFlag.active?(:sync_next_cycle) && !year == ::RecruitmentCycle.next_year
```

to avoid triggering the job twice when next year is passed in as the year argument

Reply:
yea agree 👍

Reply:
> Do we need to support this any more anyway?

I thought we would need it for Sandbox, but it would simplify lots if we could drop it!

Reply:
Discussed with @cwrw , and agreed that we don't need the flag anymore. We can reinstate it if we need it next cycle, but for now providers have no way of seeing 2022 courses even if they are synced early